### ISSUE TITLE
Crash on Android 9.0.1 with post_event

### ISSUE THREAD

**Issue by alexandrecurtelin** [defold/extension-facebook#22]
I'm building for Android 9.0.1 using defold 1.2.174

`if facebook then
		print(facebook.PARAM_LEVEL)
		local params = { [facebook.PARAM_LEVEL] = 23 }
		facebook.post_event(facebook.EVENT_ACHIEVED_LEVEL, 23, params)
end`

The code crashes the app with the following dump

`JNI DETECTED ERROR IN APPLICATION: obj == null
java_vm_ext.cc:542]     in call to CallVoidMethodV
java_vm_ext.cc:542] "Thread-49" prio=10 tid=16 Runnable
java_vm_ext.cc:542]   | group="main" sCount=0 dsCount=0 flags=0 obj=0x1ab406f0 self=0x7cbf87a000
java_vm_ext.cc:542]   | sysTid=1206 nice=-10 cgrp=default sched=0/0 handle=0x7cc5e914f0
java_vm_ext.cc:542]   | state=R schedstat=( 851708337 31170304 218 ) utm=73 stm=12 core=4 HZ=100
java_vm_ext.cc:542]   | stack=0x7cc5d94000-0x7cc5d96000 stackSize=1017KB
java_vm_ext.cc:542]   | held mutexes= "mutator lock"(shared held)
java_vm_ext.cc:542]   native: #00 pc 00000000003d0bcc  /system/lib64/libart.so (art::DumpNativeStack(std::__1::basic_ostream<char, std::__1::char_traits<char>>&, int, BacktraceMap*, char 
java_vm_ext.cc:542]   native: #01 pc 00000000004adf9c  /system/lib64/libart.so (art::Thread::DumpStack(std::__1::basic_ostream<char, std::__1::char_traits<char>>&, bool, BacktraceMap*, 
java_vm_ext.cc:542]   native: #02 pc 00000000002f27d8  /system/lib64/libart.so (art::JavaVMExt::JniAbort(char const*, char const*)+968)
java_vm_ext.cc:542]   native: #03 pc 000000000034d900  /system/lib64/libart.so (art::JNI::CallVoidMethodV(_JNIEnv*, _jobject*, _jmethodID*, std::__va_list)+1432)
java_vm_ext.cc:542]   native: #04 pc 00000000000699cc  .so (_JNIEnv::CallVoidMethod(_jobject*, _jmethodID*, 
java_vm_ext.cc:542]   native: #05 pc 000000000008b948  .so (dmFacebook::Facebook_PostEvent(lua_State*)+588)
java_vm_ext.cc:542]   native: #06 pc 00000000001dfb28  .so (???)
java_vm_ext.cc:542]   native: #07 pc 00000000001d0b14  .so (lua_pcall+124)`

**Comment by britzl** [comment]
And is this with the latest released version of extension-facebook?

**Comment by alexandrecurtelin** [comment]
Yes, using the following https://github.com/defold/extension-facebook/archive/2.6.0.zip

**Comment by britzl** [comment]
@alexandrecurtelin I am unable to reproduce this with the most recent version of the extension.

**Comment by britzl** [comment]
@alexandrecurtelin Can you please confirm if this is still a problem? I'll close this ticket otherwise.
